lte/gateway/python/magma/subscriberdb/main.py

```diff
@@ -28,7 +28,7 @@
 
 def main():
     """ main() for subscriberdb """
-    service = MagmaService('subscriberdb', mconfigs_pb2.SubscriberDB())
+    service = MagmaService('subscriberdb', mconfigs_pb2.SubscriberDB(), None, 1)
```

Comment:
We should aim to be as descriptive as we can for added parameters. 
`service = MagmaService('subscriberdb', mconfigs_pb2.SubscriberDb(), loop=None, workers=1)`